dwave/system/samplers/clique.py

```diff
@@ -126,11 +109,95 @@ def properties(self):
         return self.properties
 
     @property
-    def largest_clique_size(self):
-        """The maximum number of variables that can be embedded.
-        """
+    def largest_clique_size(self) -> int:
+        """The maximum number of variables that can be embedded."""
         return len(self.largest_clique())
 
+    @property
+    def qpu_linear_range(self) -> Tuple[float, float]:
+        """Range of values possible for the qubit biases (linear biases),
+        h, for the QPU.
```

Comment:
We'd usually keep the first line short, something like, "Range of qubit (linear) biases, h"

Reply:
Felt like it should be consistent with the system docs.